src/tools/feature-options.h

```diff
@@ -83,6 +83,11 @@ struct FeatureOptions : public Options {
              [this](Options *o, const std::string& arguments) {
                passOptions.features.setSIMD(false);
              })
+        .add("--no-validation", "-n", "Disables validation, assumes inputs are correct",
```

Comment:
It seems odd that this should be a feature option. Is it only here because you want to have it in more than one tool?

Reply:
Yeah. I agree it's a little odd - this file now means "options for all the tools". I'll rename.